### DISCUSSION TITLE
Multiple Images to Multi-frame DICOM

### DISCUSSION THREAD

**Discussion by kkmehta03** [dcm4che/dcm4che#888] [Q&A] [answered]
Hello! 
Is it possible to have a `jpg2dcm` extension so that I point it to a folder or zip containing multiple images, and that gets converted into a single multi-frame dicom?
I'm very new to this area & while browsing around this tool, I couldn't know if it already does that. 
Thanks in advance!

**Comment by vrindanayak** [comment, marked as answer]
[jpg2dcm tool](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-tool/dcm4che-tool-jpg2dcm/README.md) currently supports to convert multiple bulkdata files in directories to corresponding multiple DICOM files. 

There's no possibility in the tool to convert multiple images into single multi-frame DICOM object.

**Reply by coffLi** [reply]
have other way can convert multiple images into single multi-frame DICOM object? thank u
